docs/users.md

```diff
@@ -0,0 +1,88 @@
+# Create a Database User #
+
+You can create a MongoDB database user to authenticate to your MongoDB resource using [SCRAM](https://docs.mongodb.com/manual/core/security-scram/). First, [create a Kubernetes secret](#create-a-user-secret) for the new user's password. Then, [modify and apply the MongoDB CRD](#modify-the-mongodb-crd).
+
+You cannot disable SCRAM authentication.
+
+## Create a User Secret
+
+1. Copy the following example secret.
+
+     ```
+     ---
+     apiVersion: v1
+     kind: Secret
+     metadata:
+       name: <db-user-secret>  # corresponds to spec.users.passwordSecretRef.name in the MongoDB CRD
+     type: Opaque
+     stringData:
+       password: <my-plain-text-password> # corresponds to spec.users.passwordSecretRef.key in the MongoDB CRD
+     ...
+     ```
+1. Update the value of `metadata.name` with any name for this secret.
+1. Update the value of `stringData.password` with the user's password.
+1. Save the secret with a `.yaml` file extension.
+1. Apply the secret in Kubernetes:
+   ```
+   kubectl apply -f <db-user-secret>.yaml --namespace <my-namespace>
+   ```
+
+## Modify the MongoDB CRD
```

Comment:
Ah! Sorry I think this may have been my fault, in this case CRD refers to a [kubernetes CRD](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) which is what we provide, the user will modify a MongoDB resource definition, but not the CRD itself. I think maybe most straight forward way to phrase this is `Modify the MongoDB Resource`

I think `MongoDB resource` is a good term to refer to individual instances of the `Custom Resource` which are described via the `Custom Resource Definition (CRD)`

Reply:
Ah, I think I wasn't quite clear on that distinction myself. Thanks for the explanation! Updated.